docs/source/create_channel/create_channel_participation.md

```diff
@@ -0,0 +1,550 @@
+
+```
+Profiles:
+    SampleAppChannelEtcdRaft:
+        <<: *ChannelDefaults
+        Orderer:
+            <<: *OrdererDefaults
+            OrdererType: etcdraft
+            Organizations:
+                - <<: *SampleOrg
+                  Policies:
+                      <<: *SampleOrgPolicies
+                      Admins:
+                          Type: Signature
+                          Rule: "OR('SampleOrg.member')"
+        Application:
+            <<: *ApplicationDefaults
+            Organizations:
+                - <<: *SampleOrg
+                  Policies:
+                      <<: *SampleOrgPolicies
+                      Admins:
+                          Type: Signature
+                          Rule: "OR('SampleOrg.member')"
+
+```
+
+**Note:** For simplicity, this snippet assumes that the peers and orderers belong to the same organization `SampleOrg`. You can refer to the [sample config](https://github.com/hyperledger/fabric/blob/{BRANCH}/sampleconfig/configtx.yaml) for the full example. For a production deployment however, it is recommended that the peer and ordering nodes belong to separate organizations.
+
+An unlimited number of profiles can be listed in the `Profiles` section according to the needs of your use cases. However, each must have a distinctive name (this will not become the name of the channel itself, as that is specified by a flag given when the channel is created). In the next step, you specify which profile to use when you generate the channel genesis block.
+
+### Generate the genesis block
+
+After you have completed editing the `configtx.yaml`, you can start to create a new channel for the peer organizations. Every channel configuration starts with a genesis block. Because we previously set the environment variables for the `configtxgen` tool, you can run the following command to build the genesis block for `channel1` using the `SampleAppChannelEtcdRaft` profile:
+
+```
+configtxgen -profile SampleAppChannelEtcdRaft -outputBlock ./channel-artifacts/channel1.tx -channelID channel1
+```
+
+Where:
+- `-profile` -  Name of the profile in `configtx.yaml` that will be used to create the channel.
+- `-outputBlock` - Where to store the generated configuration block file.
+- `-channelID` - Name of the channel being created. Channel names must be all lower case, less than 250 characters long and match the regular expression ``[a-z][a-z0-9.-]*``. The command uses the `-profile` flag to reference the `SampleAppChannelEtcdRaft:` profile from `configtx.yaml`.
+
+When the command is successful, you will see logs of `configtxgen` loading the `configtx.yaml` file and printing a channel creation transaction:
+```
+INFO 001 Loading configuration
+INFO 002 orderer type: etcdraft
+INFO 003 Loaded configuration: /fabric/config/configtx.yaml
+INFO 004 Generating genesis block
+INFO 005 Writing genesis block
+
+```
+
+## Step two: Use the `osnadmin` CLI to start the channel
```

Comment:
Do we really want to use the verb "start"? Maybe that decision was already made...

I think of it as "adding the first OSN to a channel" rather than "starting" the channel. Although, it is described better in the next paragraph.

Reply:
I also made this comment at the anchor, but at the top of the document, this is referred to as 'creating' the channel, which is less correct than 'start' to me.  But I agree with Dave... this is simply joining the first node.